go-controller/pkg/ovn/egressip_test.go

```diff
@@ -3169,6 +3170,8 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 				_, err = fakeOvn.fakeClient.EgressIPClient.K8sV1().EgressIPs().Update(context.TODO(), eIPUpdate, metav1.UpdateOptions{})
 				gomega.Expect(err).ToNot(gomega.HaveOccurred())
 				fakeOvn.patchEgressIPObj(node2Name, updatedEgressIP.String())
+
+				expectedNAT.ExternalIP = updatedEgressIP.String()
```

Comment:
hmm so the egressIP was being updated but I was using the same NAT for verification?? how stupid of me... thanks @jcaamano for catching this.